Database/Corrections/Automatic/sodBaseQuests.lua

```diff
@@ -405,6 +405,94 @@ function SeasonOfDiscovery:LoadBaseQuests()
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.MAGE,
         },
+        [78192] = {
+            [questKeys.name] = "Secrets of the Light",
+            [questKeys.startedBy] = {{211188}},
+            [questKeys.finishedBy] = {{211188,}},
+            [questKeys.requiredLevel] = 17,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Kill 7 Forsaken Herbalists and 9 Forsaken Seekers in northwest Ashenvale, then return to Maethra Slagheart in Darnassus."},
+            [questKeys.objectives] = nil,
+        },
+        [78193] = {
+            [questKeys.name] = "Secrets of the Light",
+            [questKeys.startedBy] = {{211188}},
+            [questKeys.finishedBy] = {{376,}},
+            [questKeys.requiredLevel] = 17,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Travel to Stormwind City and speak to High Priestess Laurena in the Cathedral of Light."},
+            [questKeys.objectives] = nil,
+        },
+        [78194] = {
+            [questKeys.name] = "Secrets of Elune",
+            [questKeys.startedBy] = {{11397}},
+            [questKeys.finishedBy] = {{11397,}},
+            [questKeys.requiredLevel] = 17,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Collect 10 Dire Wolf Paws from wolves in Duskwood."},
+            [questKeys.objectives] = {nil,nil,{{210053}}},
+        },
+        [78195] = {
+            [questKeys.name] = "Secrets of Elune",
+            [questKeys.startedBy] = {{11397}},
+            [questKeys.finishedBy] = {{211298,}},
+            [questKeys.requiredLevel] = 17,
+            [questKeys.questLevel] = 20,
+            [questKeys.requiredRaces] = raceIDs.ALL_ALLIANCE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Bring the phial to Syrnoya at the Auberdine moonwell."},
+            [questKeys.objectives] = nil,
+        },
+        [78196] = {
+            [questKeys.name] = "Secrets of Undeath",
+            [questKeys.startedBy] = {{211229}},
+            [questKeys.finishedBy] = nil,
```

Comment:
```suggestion
            [questKeys.finishedBy] = {{211229,}},
```

Reply:
Can you elaborate this on? Is there new wowhead data for finisher? Also since I already added this as a quest fix.

Trying to understand your way of thought to make PRs that don't need changing in the future

Reply:
Yeah, I ran the scraper and checked on this quest. It returned an entry for me, so I guess the data simply changed since your commit 🤷🏻